gamla/graph_async.py

```diff
@@ -93,3 +93,61 @@ async def get_neighbors_limiting_radius(
         key=functional_generic.compose_left(functional.head, key),
     ):
         yield functional.head(s)
+
+
+_Node = Any
+
+
+@currying.curry
+async def _async_graph_traverse_many_inner(
+    seen: Set[_Node],
+    get_neighbors: Callable[[_Node], Awaitable[Iterable[_Node]]],
+    mapper: Callable[[_Node], Any],
```

Comment:
the typing here should return `None` (we don't do anything with the results)
so maybe `mapper` is not the best name.

Reply:
yes was debating that myself (the name and the typing). so maybe we should call it `add_node` ?

Reply:
`process_node` or `side_effect` maybe?